zebROS_ws/src/controller_node/launch/2023_compbot_combined.launch

```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0"?>
```

Comment:
Aren't these already merged into master?

Reply:
They are, I rebased when they were not I think, now at least I get to move some stuff over the 2023 files.